app/HutchManager/Services/RquestTaskApiClient.cs

```diff
@@ -11,26 +11,23 @@
 
 namespace HutchManager.Services
 {
-    public class RquestConnectorApiClient
+    public class RquestTaskApiClient
     {
         private readonly HttpClient _client;
-        private readonly ILogger<RquestConnectorApiClient> _logger;
-        private readonly RquestConnectorApiOptions _apiOptions;
-        private readonly IConfiguration _configuration;
+        private readonly ILogger<RquestTaskApiClient> _logger;
+        private readonly RquestTaskApiOptions _apiOptions;
 
-        public RquestConnectorApiClient(
+        public RquestTaskApiClient(
             HttpClient client,
-            ILogger<RquestConnectorApiClient> logger,
-            IOptions<RquestConnectorApiOptions> apiOptions,
-            IConfiguration configuration)
+            ILogger<RquestTaskApiClient> logger,
+            IOptions<RquestTaskApiOptions> apiOptions)
         {
           
             _client = client;
             _logger = logger;
             _apiOptions = apiOptions.Value;
-            _configuration = configuration;
             
-            string credentials = _configuration["RQUEST_USER"] + ":" + _configuration["RQUEST_PASSWORD"];
+            string credentials = _apiOptions.Username + ":" + _apiOptions.Password;
```

Comment:
⛏️ C# has string interpolation / template strings, so you can write this as `$"{_apiOptions.Username}:{_apiOptions.Password}"`